code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1051,14 +1051,13 @@
 	reagent_state = SOLID
 	taste_description = "iron"
 	pH = 6
-	overdose_threshold = 30
 	color = "#c2391d"
 
 /datum/reagent/iron/on_mob_life(mob/living/carbon/C)
 	if((HAS_TRAIT(C, TRAIT_NOMARROW)))
 		return
 	if(C.blood_volume < (BLOOD_VOLUME_NORMAL*C.blood_ratio))
-		C.blood_volume += 0.01 //we'll have synthetics from medbay.
```

Comment:
As commented, and discussed in public. hardly an 'undocumented change'.

Reply:
Someone going through files changed and pointing it out on the PR is not the kind of criteria I'd want honestly but sure, my fault for not looking closer.